### PR TITLE
Set up base components

### LINKED ISSUE #48: Structure: set up base components
Restructure project as follows.

**Components:**

- Header
	- Brand (logo and site title)
	- NavBar
	- SearchForm
- Main
	- Aside (sidebar, used on some pages as shown in wireframes)
	- Article (content of a single page or post)
	- PostGrid (grid of posts, used on News page or search results)
		- PostPreview (single post preview in grid)
- Footer
	- ContactInfo
	- SocialMedia
	- Funders

**Layouts:**

- Default
- Error

**Pages:**

Reorganize these as requested in https://github.com/inclusive-design/wecount.inclusivedesign.ca/issues/39. All of them include the same content structure, except for the News page, the Tag pages and the Search page:

```
<div id="container">
	<h1 id="title">{{ Title }}</h1>
	<div id="api-content" v-html="Content" />
</div>
```

So you should import this from the Article component.

For News, Tag, and Search, you should import the PostGrid component.

Please let me know if you have any questions about this and we can talk through it.

### PR DESCRIPTION
**Pull request by TeddTech**
resolves #48

### CODE REVIEW

File: components/Footer.vue [hunk 1]
```diff
@@ -21,6 +21,9 @@
 	</b-row>
 </template>
 <script>
+import ContactInfo from "~/components/ContactInfo"
+import SocialMedia from "~/components/SocialMedia"
+import Funders from "~/components/Funder"
```
Comment (greatislander) [inline components/Footer.vue]: I think you need to change this to "~components/Funders".

File: components/Header.vue [hunk 2]
```diff
@@ -46,6 +46,9 @@
 // TODO
 // Want to make this more of a modal
 import Aside from "~/components/Aside"
```
Comment (greatislander) [inline components/Header.vue]: Remove the `Aside` component.

File: components/Footer.vue (hunk 1 above)
Comment (TeddTech) [reply components/Footer.vue]: `~/components/Funder` is the correct way to reference the address of a component.

File: components/Header.vue [hunk 3]
```diff
@@ -45,7 +45,9 @@
 <script>
 // TODO
 // Want to make this more of a modal
-import Aside from "~/components/Aside"
+import Brand from "~/components/Brand"
+import NavBar from "~/components/NavBar"
+import SearchForm from "~/components/SearchForm"
 export default {
 	components: {
 		Aside
```
Comment (jobara) [inline components/Header.vue]: @TeddTech I notice that you aren't importing Aside any more, but there is a tag and this reference to it still. Also not sure if this line is valid. If these are errors, could you please file a new Issue to clean them up.

Comment (greatislander) [reply components/Header.vue]: @jobara This is resolved in https://github.com/inclusive-design/wecount.inclusivedesign.ca/pull/52/files

**Pull request merged**